### ISSUE TITLE
Lunamo Mouseover alt modifiers

### ISSUE THREAD

**Issue by lvangh** [Aviana/LunaUnitFrames#402]
Hello, 

I'm using /lunamo in this fashion. 

/lunamo Flash Heal(Rank 4)

However, I'm out of macro spots because I have downranked version of these spells on separate buttons bound to other keys. 

e.g. 3 is flash heal, and alt+3 is downranked flash heal. 

This is what I'm currently using specifically:
```
/stopcasting
/script if nil then CastSpellByName("Flash Heal(Rank 7)")end
/lunamo Flash Heal(Rank 7)
```

But what I need (if you can understand my logic - which doesn't work  as I just made it up) is something like this:
```
/stopcasting
/cast [mod:alt]
/script if nil then CastSpellByName("Flash Heal(Rank 3)")end
/lunamo Flash Heal(Rank 3);
/script if nil then CastSpellByName("Flash Heal(Rank 7)")end
/lunamo Flash Heal(Rank 7)
```

Basically, the alt does rank 3 and not using alt does rank 7. etc. 

Is this something like this that will work or can be added?

/lunamo [modifier:alt] code

**Comment by missesjones** [comment]
Two options I see:

1. Use individual keybindings instead. Bind R3 to Alt-<KEY>, and bind R7 to <KEY>.  
2. Use `IsAltKeyDown()` from the vanilla api.

**Comment by Aviana** [comment]
```
/stopcasting
/script if nil then CastSpellByName("Flash Heal(Rank 3)")end
/script if IsAltKeyDown() then LunaUF:Mouseover("Flash Heal(Rank 3)") else LunaUF:Mouseover("Flash Heal(Rank 3)") end
```

You can only have the tooltip for one of the ranks but this should at least give you the functionality.
